e2etest/pkg/executor/executor.go

```diff
@@ -58,7 +61,10 @@ func ForPod(namespace, name, container string) Executor {
 }
 
 func (p *podExecutor) Exec(cmd string, args ...string) (string, error) {
+	if Kubectl == "" {
+		return "", errors.New("the kubectl parameter is not set")
+	}
 	fullargs := append([]string{"exec", p.name, "-n", p.namespace, "-c", p.container, "--", cmd}, args...)
-	out, err := exec.Command("kubectl", fullargs...).CombinedOutput()
+	out, err := exec.Command(Kubectl, fullargs...).CombinedOutput()
```

Comment:
no changes asked, just question: having   exec in k8s's pod by using go client would that also also an option?

Reply:
yep, but the previous implementation was using kubectl, and that avoids bringing in extra dependencies